config/default.yml

```diff
@@ -770,6 +770,8 @@ Style/SpaceAroundOperators:
 
 Style/SpaceBeforeBlockBraces:
   EnforcedStyle: space
+  # When true, ->{ ... } is allowed.
+  ExceptAfterStabbyLambda: false
```

Comment:
Or how about `EnforcedStyleAfterStabbyArrowOperator: space` so a choice must be made, rather than accepting either space or no space.